glib-macros/tests/object_subclass_dynamic.rs

```diff
@@ -374,10 +347,8 @@ pub mod plugin {
             type Interface = MyPluginInterfaceClass;
         }
 
-        pub trait MyPluginInterfaceImpl: ObjectImpl + ObjectSubclass
-        where
-            <Self as ObjectSubclass>::Type: IsA<glib::Object>,
-            <Self as ObjectSubclass>::Type: IsA<super::MyPluginInterface>,
+        pub trait MyPluginInterfaceImpl:
```

Comment:
Should do the same for all the other extension traits I guess. You look into this afterwards?

Reply:
I just added the comment at a random place. I mean for example `pub trait FileExt: IsA<File> + sealed::Sealed + 'static` could lose the sealing AFAICS, what we talked about on matrix 2 weeks or so ago

Reply:
That kind of change should be easy. But I can also take a look at it one of these days :)

Reply:
if there is something that needs to be done on gir side, just let me know what is the change you want to have. I am a bit familiar with the codebase already, no need for extra-suffering ;)

Reply:
@bilelmoussaoui Instead of having the sealing it is sufficient for the extension traits to require `IsA<TheType>`

Reply:
And when regenerating, the sealing also should be dropped from the manual extension traits like that. I hope that slightly improves compile times too :)